coriolis/tests/conductor/rpc/test_server.py

```diff
@@ -22,6 +22,7 @@
 from coriolis.tests import testutils
 from coriolis import utils
 from coriolis.worker.rpc import client as rpc_worker_client
+from coriolis.conductor.rpc.server import ConductorServerEndpoint
```

Comment:
This is already imported above, as `server`. Please refer to the class as `server.ConductorServerEndpoint`. 
We never directly import classes, only whole modules. We generally follow openstack code style guidelines, so please also follow them: https://docs.openstack.org/hacking/latest/user/hacking.html